Doc/howto/argparse.rst

```diff
@@ -788,6 +788,59 @@ but not both at the same time:
      -q, --quiet
 
 
+Internationalization support
+============================
+
+The output of the :mod:`argparse` module such as its help text and error
+messages are all made translatable using the :mod:`gettext` module. This
+allows applications to easily localize messages produced by
+:mod:`argparse`. (If you are not familiar with Internationalization and
```

Comment:
I'd prefer just: "See also: <other docs>" - it's simpler to read, and the user can decide.